pytorch_lightning/accelerators/accelerator.py

```diff
@@ -338,21 +338,42 @@ def lightning_module_state_dict(self) -> Dict[str, Union[Any, Tensor]]:
         return self.training_type_plugin.lightning_module_state_dict()
 
     def barrier(self, name: Optional[str] = None) -> None:
+        """
+        .. deprecated:: v1.5
+            This method is deprecated in v1.5 and will be removed in v1.6.
+            Please call training_type_plugin.barrier directly
+        """
+        rank_zero_deprecation(
+            "This method is deprecated in v1.5 and will be removed in v1.6."
```

Comment:
```suggestion
            "This method is deprecated in v1.5 and will be removed in v1.6. "
```

The other deprecation messages are also missing the whitespace